src/sentry/api/endpoints/organization_release_assemble.py

```diff
@@ -69,19 +82,19 @@ def post(self, request: Request, organization, version) -> Response:
         if not chunks:
             return Response({"state": ChunkFileState.NOT_FOUND, "missingChunks": []}, status=200)
 
-        set_assemble_status(
-            AssembleTask.RELEASE_BUNDLE, organization.id, checksum, ChunkFileState.CREATED
-        )
+        set_assemble_status(assemble_task, organization.id, checksum, ChunkFileState.CREATED)
 
         from sentry.tasks.assemble import assemble_artifacts
 
         assemble_artifacts.apply_async(
             kwargs={
                 "org_id": organization.id,
                 "version": version,
+                "project_ids": project_ids,
+                "dist": None,  # TODO: a release can have multiple dists, which one is this?
```

Comment:
Well, this is a problem and I have no idea about this. Right now the lookup traverses the dists tied to a release in order to fetch the correct bundle, thus the correct behavior would be to iterate over all dists and index accordingly but this is a pain in the ass.

Reply:
I investigated this, and the existing `ReleaseFile` / artifact-index related code is using the dist embedded inside the file manifest, which is being backfilled in the assemble task. I added a NOTE, as things will work fine that way, and there is nothing else to do here.